lib/service/Dialog.dart

```diff
@@ -1067,6 +1068,93 @@ Dialog transactionConfirmed(BuildContext context) {
           ))));
 }
 
+Dialog transactionConfirmed2(BuildContext context) {
```

Comment:
Please change the name of this dialog

Reply:
I agree here, would like to see names that more clearly define the use. Furthermore using a number in the title is definitely not something we would allow in this project. I'll add this to the conventions readme